lib/apartment.rb

```diff
@@ -8,6 +8,7 @@
 
 # require_relative 'apartment/arel/visitors/postgresql'
 
+require_relative 'apartment/active_record/log_subscriber' if ActiveRecord.version.release >= Gem::Version.new('5.0')
```

Comment:
The gem version check is not needed because i've already removed the support for lower than 5.